src/wp-includes/class-wp-user-query.php

```diff
@@ -1010,6 +1023,49 @@ protected function parse_orderby( $orderby ) {
 		return $_orderby;
 	}
 
+	/**
+	 * Generate cache key.
+	 *
+	 * @since 6.2.0
+	 *
+	 * @global wpdb $wpdb WordPress database abstraction object.
+	 *
+	 * @param string $sql  SQL statement.
+	 *
+	 * @return string Cache key.
+	 */
+	protected function generate_cache_key( array $args, $sql ) {
```

Comment:
Does this need to account for the wpdb placeholder?

Reply:
It does now!